tests/unit/test.program.js

```diff
@@ -242,13 +246,37 @@ describe('program.Program', () => {
       });
   });
 
+  it('checks for updates automatically', () => {
+    const root = path.join(__dirname, '..', '..');
+    const pkgFile = path.join(root, 'package.json');
+    const handler = spy();
+    const checkForAutomaticUpdates = sinon.stub();
+    const program = new Program(['run'])
+      .command('run', 'some command', handler);
+    return execProgram(program, {
+      checkForUpdates: checkForAutomaticUpdates,
+    })
+      .then(() => {
+        assert.equal(checkForAutomaticUpdates.firstCall.args[0].name,
+                      'web-ext');
+        return fs.readFile(pkgFile);
+      })
+      .then((pkgData) => {
+        assert.equal(checkForAutomaticUpdates.firstCall.args[0].version,
+                          JSON.parse(pkgData).version);
```

Comment:
nit: this should line up with the opening parenthesis of the `assert.equal()` call